realm/build_wolfssl_with_realm.sh

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
+#bash -x ./build_wolfssl_with_realm.sh
+
+
+    # While the support to build from a tarball is included,
+    # Please note that to successfully build,
+    # you will need to manually download and set up Catch2 to match the Git
+    # repository structure when using the tarball. For example,
+        # mkdir -p test/external/generated/catch2
+        # curl -L -O
+        # https://github.com/catchorg/Catch2/archive/refs/tags/v2.13.9.tar.gz
+        # tar -xvf v2.13.9.tar.gz --strip-components=1 -C
+        # test/external/generated/catch2
+
+
+# Commit hashes for specific versions when using git
+WOLFSSL_COMMIT="e814d1ba"
+#REALM_CORE_COMMIT="c729fc80"
+REALM_CORE_COMMIT="a5e87a39"  # Adjust if necessary
+
+# Variables
+WOLFSSL_VERSION="v5.7.2-stable"
```

Comment:
Any reason to not make this "master"? I tested fine with `v5.7.4-stable`.

Reply:
I have verified that it also works with the master branch but It's best to test against a released version (default) that a customer can use.